examples/fiber/main.go

```diff
@@ -0,0 +1,134 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/airbrake/gobrake/v5"
+	fiberbrake "github.com/airbrake/gobrake/v5/fiber"
+	"github.com/gofiber/fiber/v2"
+)
+
+var ProjectId int64 = 999999                               // Insert your Project Id here
+var ProjectKey string = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx" // Insert your Project Key here
+
+var notifier = gobrake.NewNotifierWithOptions(&gobrake.NotifierOptions{
+	ProjectId:   ProjectId,
+	ProjectKey:  ProjectKey,
+	Environment: "production",
+})
+
+// Define models
+
+type Date struct {
+	Date int64 `json:"date,omitempty"`
+}
+
+type WeatherInfo struct {
+	Lat            float32       `json:"lat,omitempty"`
+	Lon            float32       `json:"lon,omitempty"`
+	Timezone       string        `json:"timezone,omitempty"`
+	TimezoneOffset float32       `json:"timezone_offset,omitempty"`
+	Current        interface{}   `json:"current,omitempty"`
+	Minutely       []interface{} `json:"minutely,omitempty"`
+	Hourly         []interface{} `json:"hourly,omitempty"`
+	Daily          []interface{} `json:"daily,omitempty"`
+}
+
+func main() {
+
+	app := fiber.New()
+	app.Use(fiberbrake.New(notifier))
+	// Initialise routes
+	app.Get("/date", getDate)
+	app.Get("/locations", getLocations)
+	app.Get("/weather/:location", getWeather)
+	app.Listen(":3000")
+	defer notifier.Close()
+	defer notifier.NotifyOnPanic()
```

Comment:
Based on our conversation, let's remove this `NotifyOnPanic` since our sample app doesn't panic.

Reply:
Done